.github/workflows/production.yml

```diff
@@ -16,37 +16,34 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - name: Write config
-      uses: schdck/create-env-json@v1
-      with:
-        file-name: 'src/config/config.json'
-        CLIENT_PASSPHRASE: ${{ secrets.WALLET_PRODUCTION_CLIENT_PASSPHRASE }}
-        CLIENT_PUBLIC_KEY: ${{ secrets.WALLET_PRODUCTION_CLIENT_PUBLIC_KEY }}
-        CLIENT_PRIVATE_KEY: ${{ secrets.WALLET_PRODUCTION_CLIENT_PRIVATE_KEY }}
-        WALLET_APP_PUBLIC_KEY: ${{ secrets.WALLET_PRODUCTION_APP_PUBLIC_KEY }}
-        WALLET_APP_AAT_SIGNATURE: ${{ secrets.WALLET_PRODUCTION_APP_AAT_SIGNATURE }}
-        POKT_USD_VALUE: ${{ secrets.POKT_USD_MARKET_PRICE }}
-        SECURE_LS_ENCRYPTION_SECRET: ${{ secrets.SECURE_LS_ENCRYPTION_SECRET }}
-        SECURE_LS_ENCODING_TYPE: "aes"
-        SECURE_LS_IS_COMPRESSION: true
-        DISPATCHERS: "https://node1.mainnet.pokt.network,https://node2.mainnet.pokt.network,https://node3.mainnet.pokt.network,https://node4.mainnet.pokt.network,https://node5.mainnet.pokt.network,https://node6.mainnet.pokt.network,https://node7.mainnet.pokt.network,https://node8.mainnet.pokt.network,https://node9.mainnet.pokt.network,https://node10.mainnet.pokt.network,https://node11.mainnet.pokt.network,https://node12.mainnet.pokt.network,https://node13.mainnet.pokt.network,https://node14.mainnet.pokt.network,https://node15.mainnet.pokt.network,https://node16.mainnet.pokt.network,https://node17.mainnet.pokt.network,https://node18.mainnet.pokt.network,https://node19.mainnet.pokt.network,https://node20.mainnet.pokt.network"
-        HTTP_PROVIDER: "https://node1.dashboard.mainnet.pokt.network:443"
-        AAT_VERSION: "0.0.1"
-        MAX_DISPATCHERS: 0
-        PROVIDER_TYPE: "http"
-        BLOCK_EXPLORER_BASE_URL: "https://explorer.pokt.network"
-        DASHBOARD_BASE_URL: "https://dashboard.pokt.network"
-        BUY_POKT_BASE_URL: "https://pokt.network/pokt-sale/"
-        CHAIN: "0001"
-        CHAIN_ID: "mainnet"
-        BLOCK_TIME: "900000"
-        MAX_TRANSACTION_LIST_COUNT: "100"
-        TX_FEE: "10000"
-        SESSION_LENGTH: "30"
     - name: NPM install
       run: npm install
     - name: Production build
       run: npm run build
+      env:
+       REACT_APP_CLIENT_PASSPHRASE: ${{ secrets.WALLET_PRODUCTION_CLIENT_PASSPHRASE }}
```

Comment:
Now that we have switched to building the application's configuration from the environment variables, the encryption keys will be exposed through the console. so I suggest to not go with env variables for this.

_PS: In all fairness, I've only went with env for config to establish conformity with the latest changes in the explorer, but not until I've finished that I have realized that this was completely automatic mode and no particular review or "motive" was behind this decision except conformity with the other repos. This makes sense for most of the variables in here, except for the secrets._

Reply:
Also if you are wondering why `REACT_APP_`:

short answer: cuz `react-scripts` and there is a small gymnastic that removes this prefix in loading time (_src/config/config.js_) so that the app is not polluted with this prefix.

Short long answer: don't wanna hassle over adding dotenv